### ISSUE TITLE
Missing home screen of Riot-Android

### ISSUE THREAD

**Issue by schildbach** [vector-im/riotX-android#1241]
I've been using Riot-Android and RiotX-Android in parallel for some time, and I must say I don't really like the experience of RiotX as much as I like Riot. It is badly missing the homescreen.

With Riot, there is an overview of all rooms (1:1 or multiuser) at the same time. I can easily see which rooms have new messages waiting. With RiotX, I miss this situational awareness: I have to navigate through all the tabs and all room names are printed in bold – regardless of wether there are new messages or not.

**Comment by promi** [comment]
I agree, in Riot I have a list of all my rooms (regardless of whether they are direct chats or group chats) in order of activity and I can just tap on the room, read messages and go back, then select the next room.

With RiotX another level is added by the room separation, so I have to go back two levels, it's just too much nesting for me.
